samples/java/build.gradle

```diff
@@ -7,6 +7,12 @@ repositories {
     mavenCentral()
 }
 
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(11)
```

Comment:
IMO, this should be Java 8 since the GE plugin still works with Java 8. Using Java 11 here might convey to the user that Java 11 is the minimum requirement.